javascript/javascript3/week2/JS3W2/js3w2.js

```diff
@@ -0,0 +1,105 @@
+// Movies exercise
+
+async function getBadMovies() {
+  try {
+    const responseMovies = await fetch(
+      "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    );
+    const movies = await responseMovies.json();
+    const badMovies = movies.filter((movie) => movie.rating < 4);
+    const badMoviesAfter = badMovies.filter((movie) => movie.year > 2000);
+    console.log(badMovies);
+    console.log(badMoviesAfter);
+  } catch (error) {
+    throw "Something went wrong";
+  }
+}
+getBadMovies();
+
+//Promise that resolves after set time
+
+function callsAfterXSeconds(num) {
+  const miliseconds = num * 1000;
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
```

Comment:
there is a small mistake here. `resolve(..)` expects an argument. For example `resolve('timeout triggered after x seconds')`

Reply:
More details
`Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)`